frontend/lib/src/components/widgets/Slider/Slider.tsx

```diff
@@ -282,159 +149,294 @@ class Slider extends React.PureComponent<Props, State> {
+    }),
+    [setValueWithSource]
   )
 
-  private renderTickBar = (): JSX.Element => {
-    const { disabled, element } = this.props
-    const { max, min } = element
+  const handleChange = useCallback(
+    ({ value }: { value: number[] }): void => {
+      setUiValue(value)
+      debouncedSetValueWithSource(value)
+    },
+    [setValueWithSource]
+  )
 
+  const renderTickBar = useCallback((): ReactElement => {
     return (
       <StyledTickBar data-testid="stSliderTickBar">
         <StyledTickBarItem
           disabled={disabled}
           data-testid="stSliderTickBarMin"
         >
-          {this.formatValue(min)}
+          {formattedMinValue}
         </StyledTickBarItem>
         <StyledTickBarItem
           disabled={disabled}
           data-testid="stSliderTickBarMax"
         >
-          {this.formatValue(max)}
+          {formattedMaxValue}
         </StyledTickBarItem>
       </StyledTickBar>
     )
-  }
-
-  public render(): React.ReactNode {
-    const { disabled, element, theme, width, widgetMgr } = this.props
-    const { colors, fonts, fontSizes, spacing } = theme
-    const style = { width }
-
-    // Manage our form-clear event handler.
-    this.formClearHelper.manageFormClearListener(
-      widgetMgr,
-      element.formId,
-      this.onFormCleared
-    )
+    // Only run this on first render. No real reason, just nicer.
+    /* eslint-disable react-hooks/exhaustive-deps */
+  }, [])
+
+  const renderThumb = useCallback(
+    /* eslint-disable react/display-name */
+    React.forwardRef<HTMLDivElement, StyleProps>(
+      (props: StyleProps, ref): ReactElement => {
+        const { $thumbIndex } = props
+        const thumbIndex = $thumbIndex || 0
+        thumbRefs[thumbIndex] = ref as React.MutableRefObject<HTMLDivElement>
+        thumbValueRefs[thumbIndex] ||= React.createRef<HTMLDivElement>()
+
+        const passThrough = pick(props, [
+          "role",
+          "style",
+          "aria-valuemax",
+          "aria-valuemin",
+          "aria-valuenow",
+          "tabIndex",
+          "onKeyUp",
+          "onKeyDown",
+          "onMouseEnter",
+          "onMouseLeave",
+          "draggable",
+        ])
+
+        const formattedValue = formattedValueArr[thumbIndex]
+
+        return (
+          <StyledThumb
+            {...passThrough}
+            disabled={props.$disabled === true}
+            ref={thumbRefs[thumbIndex]}
+            aria-valuetext={formattedValue}
+            aria-label={thumbAriaLabel}
+          >
+            <StyledThumbValue
+              data-testid="stSliderThumbValue"
+              disabled={props.$disabled === true}
+              ref={thumbValueRefs[thumbIndex]}
+            >
+              {formattedValue}
+            </StyledThumbValue>
+          </StyledThumb>
+        )
+      }
+    ),
+    // Only run this on first render, to avoid losing the focus state.
+    /* eslint-disable react-hooks/exhaustive-deps */
+    []
```

Comment:
Likely the same situation here as mentioned in the other comment about components not updating live